xbmc/interfaces/legacy/Control.h

```diff
@@ -913,7 +912,7 @@ namespace XBMCAddon
     ///
     ///
     ///-------------------------------------------------------------------------
-    /// @python_v18 Deprecated **isPassword**
+    /// @python_v19 Removed **isPassword**
```

Comment:
we need to keep the history clean, please leave the `@python_v18` deprecation note and add a new one for v19